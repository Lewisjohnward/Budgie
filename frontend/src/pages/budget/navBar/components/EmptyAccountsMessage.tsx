import clsx from "clsx";

export function EmptyAccountsMessage() {
  return (
    <div className="space-y-2">
      <div className={clsx("px-2 py-2 bg-white/10 rounded text-sm")}>
        <div className="w-56">
          <p className="font-semibold">No Accounts</p>
          <p>
            You can't budget without adding accounts to YNAB first. How about
            adding one now?
          </p>
        </div>
      </div>
    </div>
  );
}
