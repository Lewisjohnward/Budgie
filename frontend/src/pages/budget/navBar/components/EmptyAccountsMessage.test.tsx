import { render } from "@testing-library/react";
import { EmptyAccountsMessage } from "./EmptyAccountsMessage";

describe("Empty accounts message", () => {
  it("Message renders correctly", () => {
    const { asFragment } = render(<EmptyAccountsMessage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
