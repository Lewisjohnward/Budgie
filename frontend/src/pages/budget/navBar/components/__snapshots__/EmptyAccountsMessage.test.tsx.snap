// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Empty accounts message > Message renders correctly 1`] = `
<DocumentFragment>
  <div
    class="space-y-2"
  >
    <div
      class="px-2 py-2 bg-white/10 rounded text-sm"
    >
      <div
        class="w-56"
      >
        <p
          class="font-semibold"
        >
          No Accounts
        </p>
        <p>
          You can't budget without adding accounts to YNAB first. How about adding one now?
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
